src/layouts/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  footer: {
+    justifyContent: "space-evenly",
+    alignItems: "center",
+    width: "100%",
+    height: "5vh",
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    backgroundColor: "#343B41",
+    color: "#F7F7F7",
+    fontWeight: "bold",
+    textAlign: "center",
+    textTransform: "capitalize",
+  },
+  footerlink: {
+    "&:hover": {
+      textDecoration: "none",
+    }
+  }
+}));
+
+function Footer() {
+  const classes = useStyles();
+  const preventDefault = (event) => event.preventDefault();
+  const links = [
```

Comment:
const like this you should get out of the component (move to other file or move above `function Footer()` - it's some optimization: would not be created each time component rerender

Reply:
Moved link const outside.